src/templates/sections/contact.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "partials/base.njk" %}
+
+{% block meta %}
+{% endblock %}
+
+{% block content %}
+  <div class="contact-container">
+    <h1 class="contact-tite rer-font-roboto-regular">Contacto</h1>
+    <div class="contact-content">
+      <form class="contact-form">
+        <div class="contact-form-content">
+          <div class="contact-input-content margin-right-59px">
+            <label class="contact-input-label rer-font-roboto-medium" for="contact-name">Nombre</label>
+            <input class="contact-input" type="" name="" id="contact-name">
```

Comment:
Do the input tags need to have the `name` and `type` attribute  @ZhavaMan ? It's just a question.